core/models/instance.py

```diff
@@ -206,10 +209,10 @@ def get_last_history(self):
         # TODO: Profile Option
         # except InstanceStatusHistory.DoesNotExist:
         # TODO: Profile current choice
-        last_history = self.instancestatushistory_set.all().order_by(
-            '-start_date')
+        last_history = self.instancestatushistory_set.order_by(
+            '-start_date').first()
```

Comment:
Yep. this is a django thing, order_by only affects the SQL modifier at the end, if your query was going to 'work' with a call to .all, .filter, etc. then a .order_by() only changes ordering of those contents. More info here: https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.order_by